### PR TITLE
7z: add Turkish translation

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.tr/common/7z.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Yüksek sıkıştırma oranına sahip dosya sıkıştırıcısı.
+> Daha fazla bilgi: <https://www.7-zip.org/>.
+
+- Dosya veya dizin arşivle:
+
+`7z a {{archived.7z}} {{path/to/file_or_directory}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:8]: Please translate the arguments for get this easier for non-English speakers.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @Yutyo 

Remember that for translations you can send more than 1 translated page in 1 PR. This doesn't apply to the primary «English» pages.

File: pages.tr/common/7z.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Yüksek sıkıştırma oranına sahip dosya sıkıştırıcısı.
+> Daha fazla bilgi: <https://www.7-zip.org/>.
+
+- Dosya veya dizin arşivle:
+
+`7z a {{archived.7z}} {{yoldan/dosya_veya_dizine}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:8]: Please translate `archived`, I know it's not a good filename. You can use `compressed_file` for the translation, or something similar.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.tr/common/7z.md [hunk 3]
```diff
@@ -5,23 +5,23 @@
 
 - Dosya veya dizin arşivle:
 
-`7z a {{archived.7z}} {{yoldan/dosya_veya_dizine}}`
+`7z a {{sikistirilmis-dosya.7z}} {{yoldan/dosya_veya_dizine}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:8]: We use `snake_case` for variable names:
```suggestion
`7z a {{sikistirilmis_dosya.7z}} {{yoldan/dosya_veya_dizine}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.tr/common/7z.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+> Yüksek sıkıştırma oranına sahip dosya sıkıştırıcısı.
+> Daha fazla bilgi: <https://www.7-zip.org/>.
+
+- Dosya veya dizin arşivle:
+
+`7z a {{sikistirilmis_dosya.7z}} {{yoldan/dosya_veya_dizine}}`
+
+- Varolan bir arşivi çözümle (headerlar dahil):
+
+`7z a {{sifrelenmiş-dosya.7z}} -p{{parola}} -mhe=on {{archived.7z}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:12]:
```suggestion
`7z a {{sifrelenmiş_dosya.7z}} -p{{parola}} -mhe=on {{sikistirilmis_dosya.7z}}`
```

File: pages.tr/common/7z.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Varolan 7z dosyasını orijinal dizin yapısıyla dışa aktar:
+
+`7z x {{sikistirilmis-dosya.7z}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:16]:
```suggestion
`7z x {{sikistirilmis_dosya.7z}}`
```

File: pages.tr/common/7z.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Arşivi kullanıcı tarafından belirtilmiş çıkış noktasına aktar:
+
+`7z x {{sikistirilmis-dosya.7z}} -o{{yoldan/çıktıya}}`
+
+- Arşivi stdout'a aktar:
+
+`7z x {{sikistirilmis-dosya.7z}} -so`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:24]:
```suggestion
`7z x {{sikistirilmis_dosya.7z}} -so`
```

File: pages.tr/common/7z.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`7z a {{sifrelenmiş-dosya.7z}} -p{{parola}} -mhe=on {{archived.7z}}`
+
+- Varolan 7z dosyasını orijinal dizin yapısıyla dışa aktar:
+
+`7z x {{sikistirilmis-dosya.7z}}`
+
+- Arşivi kullanıcı tarafından belirtilmiş çıkış noktasına aktar:
+
+`7z x {{sikistirilmis-dosya.7z}} -o{{yoldan/çıktıya}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:20]:
```suggestion
`7z x {{sikistirilmis_dosya.7z}} -o{{yoldan/çıktıya}}`
```

File: pages.tr/common/7z.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`7z a -t{{zip|gzip|bzip2|tar}} {{archived.7z}} {{yoldan/dosya_veya_dizine}}`
+
+- Kullanılabilir arşiv türlerini sırala:
+
+`7z i`
+
+- Arşiv dosyasının içeriğini listele:
+
+`7z l {{sikistirilmis-dosya.7z}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:36]:
```suggestion
`7z l {{sikistirilmis_dosya.7z}}`
```

File: pages.tr/common/7z.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`7z x {{sikistirilmis-dosya.7z}} -o{{yoldan/çıktıya}}`
+
+- Arşivi stdout'a aktar:
+
+`7z x {{sikistirilmis-dosya.7z}} -so`
+
+- Spesifik bir arşivleme türüyle arşivle:
+
+`7z a -t{{zip|gzip|bzip2|tar}} {{archived.7z}} {{yoldan/dosya_veya_dizine}}`
```
Comment (navarroaxel) [inline pages.tr/common/7z.md:28]:
```suggestion
`7z a -t{{zip|gzip|bzip2|tar}} {{sikistirilmis_dosya.7z}} {{yoldan/dosya_veya_dizine}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]

**Pull request merged**